samples/suit/smp_transfer/boards/nrf54h20dk_nrf54h20_cpuapp.overlay

```diff
@@ -4,11 +4,14 @@
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
  */
 
+#include "../sysbuild/nrf54h20dk_nrf54h20_memory_map.dtsi"
+
 / {
 	chosen {
 		zephyr,entropy = &psa_rng;
 		zephyr,uart_mcumgr = &uart136;
 		zephyr,bt-hci-ipc = &ipc0;
+		extmem-device = &mx25uw63;
```

Comment:
I'm just wondering if the `extmem-device` should be enabled by default or only with the `FILE_PREFIX=extflash`?

Reply:
I would have to make a second copy of this file with the suffix. I could copy the common parts to some other file and `#include` in in both `nrf54h20dk_nrf54h20_cpuapp.overlay` and `nrf54h20dk_nrf54h20_cpuapp_extflash.overlay`.

Reply:
I think it is better to keep it in the current form - there is no nRF54H DK without extmem :slightly_smiling_face:

Reply:
Whoops, I didn't notice it and split it after all.